app/views/pages/contact.html.erb

```diff
@@ -35,6 +35,9 @@
     <p>
       Twitter: <a href="http://twitter.com/thepracticaldev">@thepracticaldev</a> 👻
     </p>
+    <p>
+      Report a bug: <a href="https://dev.to/security">dev.to/security</a> 🐛
```

Comment:
This is a bit of nitpick, but could you change this to "Report a vulnerability"? We want to make sure to communicate the right sort of reporting. :)